GCD/GrandCommonDivisor/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using GrandCommonDivisor.Core;
+
+namespace GrandCommonDivisor.ConsoleUI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {   
+                var first = ReadUserInput("Hello User!, Please provide the first number.");
+                var second = ReadUserInput("Please provide the second number.");
+                Console.WriteLine($"Here are your numbers: first: {first}, second: {second} .");
+                if (first != 0)
+                {
+                    var result = GCDUtils.CalculateGCD(first, second);                   
+                    Console.WriteLine($"The GCD for {first} and {second} is {result}");
+                    //var result2 = GCDUtils.CalculateGCD(7,21);
+                    //Console.WriteLine($"The GCD for MULTI is {result2}");
+                }      
+            }
+            catch (Exception ex)
```

Comment:
You don't need that catch here. Method ReadUserInput handles all exception by itself and CalculateGCD doesn't throw anything

Reply:
Right, this is corrected. thanks!